opencrvs-default.properties

```diff
@@ -102,6 +102,8 @@ javax.persistence.jdbc.url=jdbc:postgresql://${mosip.opencrvs.database.hostname}
 javax.persistence.jdbc.user=${mosip.opencrvs.database.user}
 javax.persistence.jdbc.username=${mosip.opencrvs.database.user}
 javax.persistence.jdbc.password=${mosip.opencrvs.database.password}
```

Comment:
please remove from line 101-104 as these properties not in use

please update and add the below properties
mosip.opencrvs.db.datasource.jdbc-url=